### DISCUSSION TITLE
ask for the eventalign.txt file after "nanopolish eventalign"

### DISCUSSION THREAD

**Discussion by lwlive** [GoekeLab/xpore#70] [Q&A] [answered]
In the beginning, I wanted to try xpore from the raw fast5 data, but I found that the fast5 files are 100 times bigger than the fastq files. Therefore I try to repeat the analysis from the eventalign.txt file. So could you kindly offer me tha eventalign.txt file with the email (lw_live@163.com).

**Comment by ploy-np** [comment, marked as answer]
Hi @lwlive,

The eventalign.txt files are also as huge as fast5 files. We had preprocessed the data using `xpore dataprep` already and stored in zenodo. You can download all the preprocessed data there ([https://zenodo.org/record/5103099](https://zenodo.org/record/5103099) and [https://zenodo.org/record/5103099](https://zenodo.org/record/5101752)).

From the preprocessed data, you can try `xpore diffmod` and `xpore postprocessing`.

**Reply by lwlive** [reply]
Hi，Thanks for your kind help! 
Now that  the eventalign.txt files are also very large, It is better for me to practice from the data using xpore dataprep!
Thanks   again!
